CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 ### Enhancement
 
 * `Realm.compactRealm()` works for encrypted Realms.
+*  Added `User.all()` that returnal all known Realm Object Server users.
```

Comment:
Too bad GitHub doesn't show timestamps with a resolution smaller than a minute so we could see who won that (comment) race.